.github/workflows/build.yml

```diff
@@ -164,6 +164,22 @@ jobs:
           pushd android 1> /dev/null
           ./gradlew clean build check test
         working-directory: example
+      - name: Cache node_modules
```

Comment:
I didn't find other syntax to in GH actions to avoid duping `if` statements :(

Reply:
That's fine. I forgive you 😛